src/main/webapp/i18n/en/lectureUnit.json

```diff
@@ -13,10 +13,47 @@
             },
             "releaseDate": "Release Date"
         },
+        "videoUnit": {
+            "created": "A new video unit created",
+            "updated": "Video unit updated",
+            "tooltip": "This is a Video Unit.",
+            "description": "Description",
+            "notReleasedTooltip": "Video only visible to teaching assistants and instructors. Release date:",
+            "youTube": {
+                "errorCode2": "The request contains an invalid parameter value. For example, this error occurs if you specify a video ID that does not have 11 characters, or if the video ID contains invalid characters.",
+                "errorCode5": " The requested content cannot be played in an HTML5 player or another error related to the HTML5 player has occurred.",
+                "errorCode100": "The video requested was not found. This error occurs when a video has been removed (for any reason) or has been marked as private.",
+                "errorCode101Or150": "The owner of the requested video does not allow it to be played in embedded players.",
+                "unknownError": "An error has occured when trying to start the video."
+            },
+            "createVideoUnit": {
+                "title": "Create Video Unit",
+                "name": "Name",
+                "namePlaceHolder": "Give the Video Unit a name",
+                "description": "Description",
+                "descriptionPlaceHolder": "Give the video a description",
+                "nameRequiredValidationError": "Name is required.",
+                "nameMaxLengthValidationError": "Name can be max 255 characters long.",
+                "descriptionMaxLengthValidationError": "Description can be max 255 characters long.",
+                "source": "YouTube Video Id",
+                "sourcePlaceHolder": "YouTube Video Id",
+                "sourceRequiredValidationError": "YouTube Video Id is required",
+                "urlPlaceholder": "Input a YouTube URL here for Video ID extraction",
+                "urlPatternValidationError": "URL does not match a link to a YouTube Video"
+            },
+            "editVideoUnit": {
+                "title": "Edit Video Unit"
+            },
+            "delete": {
+                "question": "Do you really want to delete the video unit <strong>{{ title }}</strong>? You can not undo this action!",
+                "typeNameToConfirm": "Please enter the name of the video unit for confirmation.",
```

Comment:
stay consistent with the wording of `video unit`. Sometimes you write it capitalized and sometimes not.

Reply:
Sometimes its in a headline and sometimes it is not. I like the capitalization in headlines

Reply:
Fixed in d092133ab497d6ee3aa612ad07beb64b01b1766d